CI/travis/check_kernel.sh

```diff
@@ -50,7 +50,7 @@ while IFS="" read -r p ; do
 	key=$(echo ${p//[[:space:],]/})
 	count=$(grep "\[$key\]" /tmp/libiio_iio_chan_type_name_spec | wc -l)
 	if [ "$count" -eq "0" ] ; then
-		echo $key missing from channel.c iio_chan_type_name_spec
+		echo "$key" missing from channel.c iio_chan_type_name_spec
```

Comment:
i going this route, maybe quote then entire string?

Reply:
fixed